rimage/depth_map.go

```diff
@@ -160,11 +176,26 @@ func ConvertImageToGray16(img image.Image) (*image.Gray16, error) {
 		return ii.Depth.ToGray16Picture(), nil
 	case *image.Gray16:
 		return ii, nil
+	case *image.Gray:
+		return gray8ToGray16(ii), nil
 	default:
 		return nil, errors.Errorf("don't know how to make image.Gray16 from %T", img)
 	}
 }
 
+// gray8ToGray16 applies an 8bit shift to the color and puts it in a image.Gray16.
+func gray8ToGray16(img *image.Gray) *image.Gray16 {
+	bounds := img.Bounds()
+	width, height := bounds.Dx(), bounds.Dy()
+	g16 := image.NewGray16(bounds)
+	for x := 0; x < width; x++ {
+		for y := 0; y < height; y++ {
+			g16.SetGray16(x, y, color.Gray16{Y: uint16(img.GrayAt(x, y).Y)})
```

Comment:
does this need to be scaled?

Reply:
The assumption is that the pixels still represent 1mm of distance, right? Or should it be that in 8bit model, every pixel represents 256 mm of distance? Potentially 8bit depth is ill-defined, and I should remove these parts from the PR.

Reply:
moot: I will just remove 8bit depth support since it is out of scope